bundle/manifests/argocd-operator.clusterserviceversion.yaml

```diff
@@ -1979,8 +1979,8 @@ spec:
   maturity: alpha
   provider:
     name: Argo CD Community
-  replaces: argocd-operator.v0.7.0
-  version: 0.9.0
+  replaces: argocd-operator.v0.9.0
```

Comment:
should be 0.10.0, but if this is still at 0.9.0 I think the 0.10.0 still needs to be synced over to master branch

Reply:
Fixed it after rebasing from latest master branch.